apps/mymedicare_cb/views.py

```diff
@@ -61,12 +62,21 @@ def authenticate(request):
 
     # Get the userinfo response object
     user_info = response.json()
+
+    # Add MBI validation info for logging.
+    sls_mbi_format_valid, sls_mbi_format_msg = is_mbi_format_valid(user_info['mbi'])
+    sls_mbi_format_synthetic = is_mbi_format_synthetic(user_info['mbi'])
+
     logger.info({
         "type": "Authentication:start",
         "sub": user_info["sub"],
+        "sls_mbi_format_valid": sls_mbi_format_valid,
+        "sls_mbi_format_msg": sls_mbi_format_msg,
+        "sls_mbi_format_synthetic": sls_mbi_format_synthetic,
```

Comment:
These validation routines are only being used for logging purposes currently. Hoping this will help with troubleshooting issues. The `sls_mbi_format_msg` will provide a clue about why the format did not validate, since we can't log the actual mbi value.